terraform-apply.sh

```diff
@@ -23,13 +23,15 @@ if [ -z "$AWS_ACCESS_KEY_ID" ] || [ -z "$AWS_SECRET_ACCESS_KEY" ]; then
 fi
 
 DIR="terraform-templates"
+S3_SSE=${S3_SSE:-true}
 
 if [ -n "$TEMPLATE_SUBDIR" ]; then
   DIR="$DIR/$TEMPLATE_SUBDIR"
 fi
 
 terraform remote config \
   -backend=s3 \
+  -backend-config="encrypt=${S3_SSE}" \
```

Comment:
Is there any reason to not just do away with the `S3_SSE` and just set as hardcoded `true`?